draft-ietf-httpbis-compression-dictionary.md

```diff
@@ -345,12 +345,12 @@ Vary: accept-encoding, available-dictionary
 IANA is asked to update the "HTTP Content Coding Registry" registry
 ({{HTTP}}) according to the table below:
 
-|--------|---------------------------------------------------------------------------------------|-------------|
-| Name   | Description                                                                           | Reference   |
-|--------|---------------------------------------------------------------------------------------|-------------|
-| br-d   | A stream of bytes compressed using the Brotli protocol with an external dictionary    | {{RFC7932}} |
-| zstd-d | A stream of bytes compressed using the Zstandard protocol with an external dictionary | {{RFC8878}} |
-|--------|---------------------------------------------------------------------------------------|-------------|
+|--------|--------------------------------------------------------------------------------------------------------------------------------|-------------|
+| Name   | Description                                                                                                                    | Reference   |
+|--------|--------------------------------------------------------------------------------------------------------------------------------|-------------|
+| br-d   | A stream of bytes compressed using the Brotli protocol with an external dictionary and a compression window of up to 16 MB.    | {{RFC7932}} |
```

Comment:
It might be better to avoid tables for this now.

(As a point on style, I find that if you need to define something like this, it is better to add an entire (short) section that includes citations and whatnot.)